config/routes.rb

```diff
@@ -20,6 +20,8 @@
     post :images, on: :collection
   end
 
+  resources :posts, except: :destroy
```

Comment:
Do we need to exclude destroy?
Don't we have a delete link in the views?

Reply:
You're right, I will fix this